.github/workflows/manual-build.yml

```
@@ -9,20 +9,31 @@ env:
   FASTLANE_XCODEBUILD_SETTINGS_RETRIES: 6 # increase timeout for xcodebuild settings. We have seen this command take a bit longer to do on CI server. 
 
 jobs:
-  deploy-app:
-    name: Re-release the latest stable build 
-    runs-on: macos-12
+  get-latest-git-tag:
+    name: Get latest git tag
+    runs-on: ubuntu-latest
     steps:
-      - name: Get latest git tag 
+      - name: Get latest git tag
         uses: octokit/request-action@v2.1.4
         id: get_latest_release
         with:
-          route: GET /repos/customerio/amiapp-reactnative/releases/latest
+          route: GET /repos/customerio/amiapp-flutter/releases/latest
```

```suggestion
          route: GET /repos/${{ github.repository }}/releases/latest
```

I suggest using this dynamic value instead of hard-coded.